src/server/serverUtils/statusBarNotifier.ts

```diff
@@ -49,18 +82,30 @@ export class StatusBarNotifier extends Disposable {
 	}
 
 	/**
-	 * @description called to notify that the server shut down.
+	 * @description called to notify that all of the servers are off
 	 */
 	public ServerOff(): void {

```

Comment:
lowercase method names, also just a style thing that doesn't really matter but it's best to be consistent